### ISSUE TITLE
[ FEATURE REQUEST ] - Better difficulty handling in Snap

### ISSUE THREAD

**Issue by RZed01** [Just-Natsuki-Team/NatsukiModDev#856]
# Overview:
By default, when playing Snap with Natsuki, the difficulty would increase regardless of win or loss. I think an improvement here would be something that resembles dynamic difficulty adjustment.

## Must haves:
-If you win the round, the difficulty will increase as normal, but here's the change:
-If Natsuki wins the round, the difficulty stays the same as in the previous round.

## Could haves:
-Maybe even decrease difficulty if Natsuki goes on a win streak?

## Is this feature request related to a problem? Please describe.
When I played nine back to back rounds of Snap with Natsuki, I would notice that the difficulty would increase with every round regardless of win or loss. There would be times where I would let Natsuki win, and then the very next round the cards are flying off the table, despite there being no reason for the increased competitiveness. I want to be able to play this game just to have fun with Natsuki (and not have to get serious), but the increasing difficulty per round made me quit before I otherwise would.

**Comment by Zhaumbie** [comment]
I agree. By the time I hit a streak of six, she's slapping cards and snapping before I can even see the cards she's putting down—and I have _fast_ reaction time. Rubber banding her reaction speed would go a long way to making it a more engaging experience
